python-avd/pyavd/_eos_designs/structured_config/underlay/port_channel_interfaces.py

```diff
@@ -25,6 +26,11 @@ class PortChannelInterfacesMixin(Protocol):
     Class should only be used as Mixin to a AvdStructuredConfig class.
     """
 
+    @cached_property
+    def _l3_port_channels_with_subinterfaces(self: AvdStructuredConfigUnderlayProtocol) -> set:
+        """Return a set of L3 Port-Channel names that have sub-interfaces."""
+        return {pc.name.split(".", maxsplit=1)[0] for pc in self.shared_utils.node_config.l3_port_channels if "." in pc.name}
```

Comment:
please avoid the short variable names. We prefer to use port_channel instead of pc to keep things easier to read and understand. The extra characters will not mean much for the bytecode :)